api/graphql/schema.graphql

```diff
@@ -22,53 +22,58 @@ enum Language {
 
 type User {
     id: ID!
-    firstName: String! @rename(attribute: "first_name")
-    lastName: String! @rename(attribute: "last_name")
     email: Email!
+    firstName: String @rename(attribute: "first_name")
+    lastName: String @rename(attribute: "last_name")
     telephone: PhoneNumber
     preferredLang: Language @rename(attribute: "preferred_lang")
+    pools: [Pool] @hasMany
+    poolCandidates: [PoolCandidate] @hasMany
 }
 
 type Pool {
     id: ID!
-    owner: User!
-    name: LocalizedString!
+    owner: User @belongsTo(relation: "user")
+    name: LocalizedString
     description: LocalizedString
-    classifications: [Classification]
-    essentialCriteria: [CmoAsset] @rename(attribute: "essential_criteria")
-    assetCriteria: [CmoAsset] @rename(attribute: "asset_criteria")
-    operationalRequirements: [OperationalRequirement] @rename(attribute: "operational_requirements")
+    classifications: [Classification] @belongsToMany
+    assetCriteria: [CmoAsset] @belongsToMany
+    essentialCriteria: [CmoAsset] @belongsToMany
+    operationalRequirements: [OperationalRequirement] @belongsToMany
+    poolCandidates: [PoolCandidate] @hasMany
 }
 
 enum PoolCandidateStatus {
     AVAILABLE
-    PLACED
+    PLACED_INDETERMINATE
+    PLACED_TERM
     NO_LONGER_INTERESTED
 }
 
 type PoolCandidate {
     id: ID!
-    pool: Pool!
-    user: User!
+    pool: Pool @belongsTo
+    user: User @belongsTo
+
+    # cmoIdentifier can be an arbitrary string used to relate this candidate to an external database.
     cmoIdentifier: ID @rename(attribute: "cmo_identifier")
+    # Expiry date for this candidate being in the pool.
+    expiryDate: Date
 
-    """
-    Employment Equity fields
-    """
     isWoman: Boolean @rename(attribute: "is_woman")
     hasDisability: Boolean @rename(attribute: "has_disability")
     isIndigenous: Boolean @rename(attribute: "is_indigenous")
     isVisibleMinority: Boolean @rename(attribute: "is_visible_minority")
 
     hasDiploma: Boolean @rename(attribute: "has_diploma")
-    languageAbility: LanguageAbility
-    locationPreferences: [WorkRegion]
-    acceptedOperationalRequirements: [OperationalRequirement]
-    expectedSalary: [SalaryRange]
-    expectedClassifications: [Classification]
-    cmoAssets: [CmoAsset]
+    languageAbility: LanguageAbility @rename(attribute: "language_ability")
+    locationPreferences: [WorkRegion] @rename(attribute: "location_preferences")
+    acceptedOperationalRequirements: [OperationalRequirement] @belongsToMany
+    expectedSalary: [SalaryRange] @rename(attribute: "expected_salary")
+    expectedClassifications: [Classification] @belongsToMany
+    cmoAssets: [CmoAsset] @belongsToMany
 
-    status: PoolCandidateStatus!
+    status: PoolCandidateStatus
 }
 
 enum LanguageAbility {
```

Comment:
1. Would it not make sense for `LanguageAbility` to be an array of either or both of **en**, **fr** since having both of those implies _BILINGUAL_?
2. Should it not be _PRAIRIES_ rather than _PRAIRIE_ (side question: why are some values provinces and others regions)?

Reply:
1. My perspective is that its slightly easier to work with a simple enum than check for multiple values in a list. And that we don't really want to consider an empty list a valid option.
2. Its short for "Prairie Region". The list of values comes from CMO: https://github.com/GCTC-NTGC/gc-digital-talent/issues/7